src/mmw/apps/modeling/views.py

```diff
@@ -216,18 +218,17 @@ def _initiate_tr55_job_chain(model_input, job_id):
 def _construct_tr55_job_chain(model_input, job_id):
     job_chain = []
 
-    current_hash = model_input['modification_hash']
-    census = model_input.get('census')
-    census_hash = None
+    if True:  # XXX
+        aoi = model_input.get('area_of_interest')
```

Comment:
The functionality to used cached results has been turned off (I think it needs to be reworked -- or at least augmented -- to account for the censuses of the modification pieces).